tfs/src/main/java/hudson/plugins/tfs/listeners/JenkinsRunListener.java

```diff
@@ -54,4 +94,24 @@ private String getStartedBy(final Run run) {
         }
         return startedBy;
     }
+
+    private JSONObject createJsonFromRun(final Run run) {
+        return new JSONObject();
+    }
+
+    private String getRunContext(final Run run) {
```

Comment:
Why is this code here workflow job specific?

Reply:
Good catch. It shouldn't. I've changed it to be applied to all jobs.